### ISSUE TITLE
Why don't ya add jslinter as a dependency in your package.json?

### ISSUE THREAD

**Issue by fusion809** [tensor5/linter-jslinter#2]
Hi,

I'm surprised this package doesn't, instead of asking users to install jslinter, merely require it as a package dependency listed in package.json. That is, with something like this added to package.json:

``` json
dependencies: {
  "jslinter": "1.5.2"
},
```

Thanks for your time,
Brenton

**Comment by tensor5** [comment]
It would definitely be easier for users, but then I should release a new version each time `jslinter` updates. Generally speaking as a Linux fan I prefer using globally installed tools or libraries, but in this case I'm still thinking about it.

**Comment by fusion809** [comment]
Appending a caret top (`^`) to before the version number of jslinter (i.e., changing `"1.5.2"` in the above package.json snippet to `"^1.5.2"`) will cause the latest available version of jslinter (provided it is at least 1.5.2) to be installed.

**Comment by fusion809** [comment]
But of course, that means only at the time that this atom package is installed will the latest version of jslinter be installed. So maybe you could add an auto-updating feature to this package? So that if a later version of jslinter is available than that installed in the node_modules subfolder of the package, it will be installed in place of the current version. If you want a specific shell command for this package to run, you could try:

``` bash
rm -rf $PATH_TO_PACKAGE/node_modules/jslinter
cd $PATH_TO_PACKAGE
apm install
```

**Comment by tensor5** [comment]
Yes, checking for updates can be an idea, if it can be done in background without slowing down the startup time.

**Comment by fusion809** [comment]
The [`autocomplete-go`](https://github.com/joefitzgerald/autocomplete-go) package installs (granted not in the background, it has a pop-up dialog asking whether to run `go get -u github.com/nsf/gocode`) the `gocode` package locally and doesn't seem to slow down startup much.
